src/scrabble.js

```diff
@@ -1,5 +1,98 @@
-function scrabble() {
+const scoreValues = {
```

Comment:
I like the use of the object to assign the letters a score. Looking at other peoples code a lot of people took this approach. Its very neat, The way I did it is a bit more obnoxious.